ui/app/search/search-ctrl.js

```diff
@@ -93,7 +93,6 @@
           model.qtext = qtext;
         }
 
-        model.qtext = model.qtext || mlSearch.getText();
```

Comment:
I think this was meant to preserve the ?q= parameter when directly hitting the search link. I think before merging this it would be best to also adjust line 21 to something like: `        model.qtext = model.qtext || $location.search().q || '';`